tools/docker/builder/openwrt/build.sh

```diff
@@ -1,12 +1,13 @@
-#!/bin/sh -e
+#!/bin/bash -e
```

Comment:
Please mention something like this very explicitly in the commit message. Not just that we changed the shell to bash, but also why.